sli/slimodule.h

```diff
@@ -47,6 +47,15 @@ class SLIModule
   virtual void init( SLIInterpreter* ) = 0;
 
   /**
+   * Reset the module. This function is currently only called when the
+   * NEST function ResetKernel is invoked.
```

Comment:
And one may begin to wonder about some of our design decisions when `slimodule` needs a `reset()` method solely to permit for `ResetKernel` in NEST ...